APE24.rst

```diff
@@ -0,0 +1,507 @@
+invite scientific peers of a study to be reviewers.
+Given that reviewers are anonymous by default, we are unable to provide
+a fully transparent review process (even though results are publicly posted
+and discussed, they are posted second-hand by the editors).
+
+Unlike scientific articles, which are static after publication, software continues to develop,
+adding features and fixing bugs; otherwise, it will become obsolete and might be
+incompatible with future versions of its dependencies (e.g., NumPy or Python).
+Thus, a process of "re-review" for Affiliated packages
+has been envisioned, but never executed because we have no resources nor infrastructure to
+re-review the 47 accepted packages to ensure that they still meet the
+Affiliated standards. Therefore, there is a need to restructure our Affiliated
+process to be sustainable. We are confident that a partnership with
+pyOpenSci is the best solution currently available.
+
+.. _pyOpenSci: https://www.pyopensci.org/
+
+What do we gain from a partnership with pyOpenSci?
+--------------------------------------------------
+
+* Larger exposure for Affiliated packages within the scientific Python ecosystem,
+  not just Astropy.
+* If they are in scope for JOSS, they can be fast-tracked through the JOSS
+  review process and get a publication via the
+  `partnership between pyOpenSci and JOSS <https://www.pyopensci.org/software-peer-review/partners/joss.html>`_.
+* pyOpenSci's objective way of initial triage will automatically weed out
+  packages that do not even meet basic standards (e.g., not installable),
+  reducing our editors' burdens.
+* We can be sure the process is consistent with other projects that have
+  also partnered with pyOpenSci.
+* We can share infrastructure with pyOpenSci, reducing our own maintenance burden.
+* We can share reviewer pool with pyOpenSci, providing our reviewers
+  opportunity to connect with peers beyond Astropy.
+* Even when a package ends up not being accepted into Affiliated but satisfies
+  basic pyOpenSci criteria, its application effort is not entirely wasted,
+  as it could still be listed under pyOpenSci (and JOSS) only without our Affiliated status.
+
+Who owns this process?
+----------------------
+
+pyOpenSci already has an established software peer review process and it is community-driven.
+It is compatible with Astropy but it is up to us to decide if a package
+reviewed and accepted by pyOpenSci also qualifies as Affiliated.
+
+Will the new process still foster a sense of community in Astropy?
+------------------------------------------------------------------
+
+Yes, we think so! We will still keep the `Astropy Affiliated Packages`_
+page, but now instead of hosting the content ourselves, we grab an RSS/XML feed
+from pyOpenSci for the packages marked as Affiliated. To keep the history,
+we will move the existing listing to a "legacy affiliation" page and keep it static.
```

Comment:
A strict reading of this is that it applies only to affiliated and not coordinates packages.  Is that intended?  (if so it should be more explicit I think)

Reply:
Correct, because so far (even before this APE), decision on whether a package becomes Coordinated or not does not go through the Affiliated review process. Lately, we had promoted Affiliated packages to Coordinates (e.g., specreduce, specutils) but Editors did not have any say in that, so Coordinated is not covered by this APE.

How do you wish this to be stated explicitly?

Reply:
How about this?

```suggestion
Yes, we think so! We will still keep the `Astropy Affiliated Packages`_
page, but now instead of hosting the content ourselves, we grab an RSS/XML feed
from pyOpenSci for the packages marked as Affiliated. To keep the history,
we will move the existing listing to a "legacy affiliation" page and keep it static.

Coordinated packages are separate from this due to their tighter integration with
the Astropy project's team structure, so they will remain listed on this page without
a connection to pyOpenSci.
```